src/Window.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./App.css";
+import FreeFormIdeas from "./FreeformIdeas";
+import OptionBar from "./OptionBar";
+import { ThemeContext, ThemeStateManagementProvider } from "./ThemeContext";
+import { IdeaContext, IdeaStateManagementProvider } from "./IdeaContext";
+
+const Window = () => {
+  return (
```

Comment:
```suggestion
const Window = () => (
```

Reply:
Changing the contexts to hooks and placing them inside window, now means I need to designate what is returned, right?

Reply:
Yes